src/03-control-flow.js

```diff
@@ -86,7 +94,16 @@ function positiveNegativeOrZero(a) {}
  *  aroundTheWorldGreeting();
  *  //> "Hello World"
  */
-function aroundTheWorldGreeting(language) {}
+function aroundTheWorldGreeting(language = 'english') {
```

Comment:
We could space out the ` else if ` from the code blocks to make it a bit cleaner